store/store.go

```diff
@@ -354,7 +354,7 @@ type sectionResults struct {
 var detailFields = getStructFields(snapDetails{})
 
 // The fields we are interested in for snap.ChannelSnapInfos
-var channelSnapInfoFields = getStructFields(snap.ChannelSnapInfo{})
+var channelSnapInfoFields = getStructFields(channelSnapInfoDetails{})
```

Comment:
@niemeyer it gets unmarshalled into a store.snapDetails struct (I have not changed this). However I think you are right, this is confusing, I changed the code to use store.channelSnapInfoDetails for this now too. Should make it more obvious and hopefully make it easier to adapt the code when chanenls 2.0 come around.